python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -415,7 +415,7 @@ class IsrTaskLSSTConfig(pipeBase.PipelineTaskConfig,
     brighterFatterApplyGain = pexConfig.Field(
         dtype=bool,
         doc="Should the gain be applied when applying the brighter-fatter correction?",
-        default=True,
+        default=False,
```

Comment:
Should this config be removed entirely?  And instead if the image is in `adu` units then the gain is applied, and if it is in `electron` units the gain is not applied?

Reply:
Sure. I'll need to review it more carefully to make sure it doesn't break anything in isrFunctions. I'll save that for the morning after I get some sleep :)

Reply:
We shouldn't need to touch anything in isrFunctions.  This is only used internally by this task, and can be inferred from the units directly.

Reply:
I've added a check of the current exposure units and determine the application of the gains internally. I then deleted the user config for` brighterFatterApplyGain`. Everything should always be applied in the correct units now.